circle.yml

```diff
@@ -41,12 +42,12 @@ test:
         background: true
         parallel: true
   override:
-    - ./gradlew --profile --continue --parallel compileJava compileTestJava:
+    - ./scripts/time-cmd.sh ./gradlew --profile --continue --parallel compileJava compileTestJava:
```

Comment:
It should be virtually 0. We're just piping the output (which is quite small) through a super small program